skimage/transform/_geometric.py

```diff
@@ -570,8 +570,30 @@ class ProjectiveTransform(GeometricTransform):
     params : (D+1, D+1) array
         Homogeneous transformation matrix.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage import data
+    >>> from skimage.transform import ProjectiveTransform
+    >>> img = data.rocket()
+    >>> height, width, dim = img.shape
+
+    Provide four pairs of matching points between the source and
+    destination images to estimate the homography matrix, H,
+    automatically for you.
+
+    >>> src = np.array([[41., 74.],
+    ...                 [width, 72.],
+    ...                 [height, width],
+    ...                 [96., height]])
+    >>> dst = np.array([[0., 0.],
+    ...                 [0., height],
+    ...                 [height-50, width*0.75],
+    ...                 [width*0.75, 80]])
+    >>> pt = ProjectiveTransform()
+    >>> pt.estimate(src, dst)
+    True
```

Comment:
Looks great! I especially like that you added meaning to the points by using width and height. How did you choose the values for `src` and `dst`? 

I think it would be nice though, if the example showed what to do with the estimated transform. E.g. warp the image or show the residuals.

Reply:
I chose the value for `src` by randomly selecting four control points in the source image.
I have since revised my implementation, with the value of  `src` matching the four corners of the input image(`img`) while the value of `dst` are arbitrary control points with plane of the source image